### PR TITLE
Refactor: Move network config to clusterconfig

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: internal/clusterconfig/clusterconfig.go [hunk 1]
```diff
@@ -47,12 +47,18 @@ const (
...
+	networkDir = "/opt/openshift/network-configuration"
 )
 
+var hostPath = common.Host
+var listOfNetworkFilesPaths = []string{
+	"/etc/hostname",
+	"/etc/NetworkManager/system-connections",
+}
+
 var (
```
Comment (pixelsoccupied) [inline internal/clusterconfig/clusterconfig.go:62]: Consider moving the new vars in the var() block

**Review by pixelsoccupied — COMMENTED** [review]
Just a tiny comment but otherwise the changes are straightforward and some nice refactoring...LGTM from me

File: internal/clusterconfig/clusterconfig.go (hunk 1 above)
Comment (tsorya) [reply internal/clusterconfig/clusterconfig.go:62]: right, missed it

**Review by donpenney — COMMENTED** [review]
/lgtm

**Pull request merged**